src/profile_manager/urls.py

```diff
@@ -21,6 +22,9 @@
     url(r'^(?P<pk>[0-9]+)/edit/$', views.ProfileUpdate.as_view(), name='profile_update'),
     url(r'^edit/own/$', views.ProfileUpdateOwn.as_view(), name='profile_edit_own'),
 
+    path('change_username/<int:pk>/', views.UsernameUpdate.as_view(), name='change_username'),
+    path('reset_password/<int:pk>/', views.PasswordReset.as_view(), name='change_password'),
```

Comment:
Maybe give this a more specific name to the password change since this is for staff to reset student passwords.  name="password_change_bystaff" or something verbose and crystal clear like that.

Also, don't use underscores in urls if you can avoid it, use dashes, or even better, match the existing authentication form: [account/]password/change

So maybe:
[profile/]username/change/
[profile/]password/change/bystaff

Reply:
changed password url to ```password/change/<int:pk>/```